### PR TITLE
[powermax] Update README.md

### PR DESCRIPTION
**Pull request by franjuan**
Added some comments on how to enable bypass switching

### CODE REVIEW

File: bundles/org.openhab.binding.powermax/README.md [hunk 1]
```diff
@@ -155,6 +155,7 @@ In case it fails after all the retries, you still have the option to later trigg
 -   Visonic does not provide a specification of the RS232 protocol and, thus, use this binding at your own risk.
 -   The binding is not able to arm/disarm a particular partition.
 -   The compatibility of the binding with the Powermaster alarm panel series is probably only partial.
+-   In order to be able to bypass zones, `Bypass` parameter at `Panel parameters` at PowerMax must be set to `manual bypass` as described in section 4.4.8 in installer's manual (Installer mode -> 4. Define Panel -> 8. Bypass).
```
Comment (lolodomo) [inline bundles/org.openhab.binding.powermax/README.md:158]: The message is too much specific to a model.
But you are right, to be able to use this feature, there is a setting to change in the alarm panel, this is the case even for a PowermaxPro.
I would not put that comment as this place as it is not a limit of the binding. I believe it should be added as a comment for the channel itself.

Comment (lolodomo) [reply bundles/org.openhab.binding.powermax/README.md:158]: I propose something like "In order to be able to bypass zones, you must first enable this feature by updating your panel configuration (installer mode)."

**Review by lolodomo — APPROVED** [review]

**Comment by franjuan** [comment]
I changed as you proposed

**Review by fwolter — APPROVED** [review]
LGTM. Next time, please sign-off your commits: https://www.openhab.org/docs/developer/contributing.html#sign-your-work Applying the small patch exception.

**Pull request merged**